src/jquery.fancytree.fixed.js

```diff
@@ -377,21 +377,21 @@
 
 		nodeRender: function(ctx, force, deep, collapsed, _recursive) {
 			var res = this._superApply(arguments),
-				node = ctx.node,
-				isRootNode = !node.parent;
+				isRootNode = !ctx.node.parent;
```

Comment:
shouldn't we leave `node = ctx.node` here, since it is used in the following line?

Reply:
In the case when isRootNode variable is true, node variable will be redundant